packages/itwinui-css/src/button/split-button.scss

```diff
@@ -2,46 +2,46 @@
 // See LICENSE.md in the project root for license terms and full copyright notice.
 @import '../style/index';
 
-@mixin iui-button-split-menu {
+@mixin iui-button-split {
   @include iui-button-group;
 
   &:hover {
     background-color: rgba(var(--iui-color-foreground-body-rgb), var(--iui-opacity-6));
   }
 
   > * {
-    &:first-child {
-      > .iui-borderless:hover {
-        background-color: transparent;
-      }
-    }
+    // &:first-child {
+    //   > [data-iui-button-variant='borderless']:hover {
+    //     background-color: transparent;
+    //   }
+    // }
```

Comment:
Unsure what lines 13-17 are used for.

Reply:
This code was for the background color for split buttons in the header.

Before:
![SplitMenuChangesBefore](https://user-images.githubusercontent.com/78564221/176757575-e31ece30-37a1-4fde-ab5d-b6624a3d9125.gif)


After: 
![SplitMenuChangesAfter](https://user-images.githubusercontent.com/78564221/176757607-58853877-47e7-4229-9a99-a4c1be60188d.gif)